app/admin/dashboard.rb

```diff
@@ -3,31 +3,21 @@
   menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
 
   content title: proc{ I18n.t("active_admin.dashboard") } do
-    div class: "blank_slate_container", id: "dashboard_default_message" do
-      span class: "blank_slate" do
-        span I18n.t("active_admin.dashboard_welcome.welcome")
-        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+
+    columns do
+      column do
+        panel "Info" do
+            @user_chapter = Chapter.where(id: current_admin_user.chapter_id)
+            @user_chapter.each do |c|
+                if c.is_active?
```

Comment:
yeah I was curious why it WAS working haha I kept switching the data to see it go and it was fine...not sure I like that feeling though so I'll put it back to just c.is_active...thanks again :+1: